inc/shortcodes/class-scribd.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Shortcake_Bakery\Shortcodes;
+
+class Scribd extends Shortcode {
+
+	public static function get_shortcode_ui_args() {
+		return array(
+			'label'          => esc_html__( 'Scribd', 'shortcake-bakery' ),
+			'listItemImage'  => 'TK',
```

Comment:
@lisaej  @danielbachhuber We can just use this one http://uxrepo.com/static/icon-sets/zocial/svg/scribd.svg